scripts/install

```diff
@@ -104,159 +48,61 @@ ynh_app_setting_set --app=$app --key=virtualisation --value=$virtualisation
+ynh_setup_source --dest_dir="$install_dir"
 
+chmod -R o-rwx "$install_dir"
+chown -R $app:$app "$install_dir"
+chmod +x $install_dir/garage
 
 #=================================================
 # NGINX CONFIGURATION
 #=================================================
 ynh_script_progression --message="Configuring NGINX web server..."  --weight=1
 
-### `ynh_add_nginx_config` will use the file conf/nginx.conf
-
 # Create a dedicated NGINX config
 ynh_add_nginx_config
 
-
 #=================================================
 # ADD A CONFIGURATION
 #=================================================
 ynh_script_progression --message="Adding a configuration file..."  --weight=1
 
-### You can add specific configuration files.
-###
-### Typically, put your template conf file in ../conf/your_config_file
-### The template may contain strings such as __FOO__ or __FOO_BAR__,
-### which will automatically be replaced by the values of $foo and $foo_bar
-###
-### ynh_add_config will also keep track of the config file's checksum,
-### which later during upgrade may allow to automatically backup the config file
-### if it's found that the file was manually modified
-###
-### Check the documentation of `ynh_add_config` for more info.
-
-	ynh_add_config --template="mount_disk.sh" --destination="$final_path/mount_disk.sh"
-	ynh_add_config --template="umount_disk.sh" --destination="$final_path/umount_disk.sh"
-	ynh_add_config --template="garage.toml" --destination="$final_path/garage.toml"
-	chmod +x "$final_path/mount_disk.sh" "$final_path/umount_disk.sh"
-
-# FIXME: this should be handled by the core in the future
-# You may need to use chmod 600 instead of 400,
-# for example if the app is expected to be able to modify its own config
-chmod 600 "$final_path/garage.toml"
-chown $app:$app "$final_path/garage.toml"
-
-### For more complex cases where you want to replace stuff using regexes,
-### you shoud rely on ynh_replace_string (which is basically a wrapper for sed)
-### When doing so, you also need to manually call ynh_store_file_checksum
-###
-### ynh_replace_string --match_string="match_string" --replace_string="replace_string" --target_file="$final_path/some_config_file"
-### ynh_store_file_checksum --file="$final_path/some_config_file"
+ynh_add_config --template="mount_disk.sh" --destination="$install_dir/mount_disk.sh"
+ynh_add_config --template="umount_disk.sh" --destination="$install_dir/umount_disk.sh"
+ynh_add_config --template="garage.toml" --destination="$install_dir/garage.toml"
+chmod +x "$install_dir/mount_disk.sh" "$install_dir/umount_disk.sh"
 
+chmod 600 "$install_dir/garage.toml"
+chown $app:$app "$install_dir/garage.toml"
 
 #=================================================
 # CREATE DATA DIRECTORY
 #=================================================
 ynh_script_progression --message="Creating a data directory..."  --weight=5
+
 if [ "$virtualisation" = "true" ]
 then
  ynh_print_warn --message="This may take time regarding disk size…"
 fi
```

Comment:
Should probably be merged with what comes after

Reply:
What do you mean ?